packages/bls12-381-hash/bls12-381-hash.0.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis:
+  "Implementation of some cryptographic hash primitives using the scalar field of BLS12-381"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-bls12-381-hash"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-bls12-381-hash/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.8.4"}
+  "bls12-381" {>= "5.0.0"}
+  "alcotest" {with-test}
+  "bisect_ppx" {with-test & >= "2.5"}
+]
+available: arch != "ppc64" & arch != "arm32" & arch != "x86_32"
```

Comment:
```suggestion
available: arch != "ppc64" & arch != "arm32" & arch != "x86_32" & arch != "s390x"
```
Should we mark it unavailable or you prefer to fix it?

Reply:
Yes, for the first version let's do it. I will work on a fix later. Thanks!